src/AzureStack/SubscriptionsAdmin/Subscriptions.Admin/generate.ps1

```diff
@@ -0,0 +1,13 @@
+# Remove Old
+Remove-Item "$PSScriptRoot\Generated" -Force -Recurse -ErrorAction SilentlyContinue
```

Comment:
I dont't think you should need these additional steps. Could you please pull from latest master and regenerate the code by running the ps1 similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1)

Reply:
fixed. Thanks